vars.yml

```diff
@@ -2,7 +2,80 @@
 # Ansible Tower Installation
 #installation_tarball: https://releases.ansible.com/ansible-tower/setup-bundle/ansible-tower-setup-bundle-latest.tar.gz
 installation_tarball: http://perf54.perf.lab.eng.bos.redhat.com/pub/ansible-automation-platform-setup-bundle-2.0.0-1-early-access.tar.gz
-inventory_template: inventory_cluster_4.j2
+
+scenario: "control-cluster-hop-to-execution-plane"
+
+aap_cluster_topology_map:
+  "control-cluster":
+    control_nodes: 3
+    hybrid_nodes: 0
+    execution_nodes: 3
+    legacy_isolated_nodes: 0
+    hop_nodes: 0
+    inventory_template: "inventory_cluster_4.1.j2"
+  "hybrid-cluster":
+    control_nodes: 0
+    hybrid_nodes: 3
+    execution_nodes: 3
+    legacy_isolated_nodes: 0
+    hop_nodes: 0
+    inventory_template: "inventory_cluster_4.1.j2"
+  "control-cluster-1-hop-to-execution-plane":
+    control_nodes: 3
+    execution_nodes: 2
+    hybrid_nodes: 0
+    legacy_isolated_nodes: 0
+    hop_nodes: 1
+    inventory_template: "inventory_cluster_4.1.j2"
+  "control-cluster-2-hop-to-execution-plane":
+    control_nodes: 3
+    execution_nodes: 2
+    hybrid_nodes: 0
+    legacy_isolated_nodes: 0
+    hop_nodes: 2
+    inventory_template: "inventory_cluster_4.1.j2"
+  "control-cluster-3-hop-to-execution-plane":
+    control_nodes: 3
+    execution_nodes: 2
+    hybrid_nodes: 0
+    legacy_isolated_nodes: 0
+    hop_nodes: 3
+    inventory_template: "inventory_cluster_4.1.j2"
+  "hybrid-cluster-hop-to-execution-plane":
+    control_nodes: 0
+    hybrid_nodes: 3
+    execution_nodes: 2
+    legacy_isolated_nodes: 0
+    hop_nodes: 1
+    inventory_template: "inventory_cluster_4.1.j2"
+  "single-hybrid":
+    control_nodes: 0
+    hybrid_nodes: 1
+    execution_nodes: 0
+    legacy_isolated_nodes: 0
+    hop_nodes: 0
+    inventory_template: "inventory_cluster_4.1.j2"
+  "legacy-cluster-with-isolated-nodes":
+    control_nodes: 0
+    hybrid_nodes: 3
+    execution_nodes: 0
+    legacy_isolated_nodes: 2
+    hop_nodes: 0
+    inventory_template: "inventory_cluster_4.j2"
+  "legacy-cluster":
+    control_nodes: 0
+    hybrid_nodes: 3
+    execution_nodes: 0
+    legacy_isolated_nodes: 0
+    hop_nodes: 0
+    inventory_template: "inventory_cluster_4.j2"
+

```

Comment:
We should do this some where else may be ? or may be just use the whole dictionary via a different vars file. This repo is used by other teams as well who just create infrastructure (vm's) with tower being installed. They just set install_tower to false like mentioned in the readme.

Reply:
I think ideally we would do 
`automationcontroller <- hop_node[0] <- hop_node[1] .... <- execution_nodes[:]`\

I'll poke at this some more

Reply:
I'm open to moving the tower stuff into a seperate vars file we pass in the jenkins job we use, but I'd move the isolated nodes and execution nodes vars too.

Reply:
True. I think at this point we should do that. I think that vars.yml is growing into a very Tower/Controller specific file, I am responsible for that. We can keep that file to bare minimum required for spinning up vm's on IBM Cloud without Tower/Controller and move Tower/Controller specific part to different file.